crates/miden-objects/src/batch/proposed_batch.rs

```diff
@@ -162,16 +163,17 @@ impl ProposedBatch {
         // this block's inclusion when including this batch and verifying its ZK proof.
         //
         // Finally, note that we don't verify anything cryptographically here. We have previously
-        // verified that the batch reference block's chain root matches the hashed peaks of the
-        // `ChainMmr`, and so we only have to check if the chain MMR contains the block here.
+        // verified that the batch reference block's chain commitment matches the hashed peaks of
+        // the `ChainMmr`, and so we only have to check if the chain MMR contains the block
+        // here.
         // --------------------------------------------------------------------------------------------
 
         for tx in transactions.iter() {
             if reference_block_header.block_num() != tx.block_num()
                 && !chain_mmr.contains_block(tx.block_num())
             {
                 return Err(ProposedBatchError::MissingTransactionBlockReference {
-                    block_reference: tx.block_ref(),
+                    block_reference: tx.block_commitment(),
```

Comment:
Not from this PR, but should `ProvenTransaction::block_commitment()` be actually `ProvenTransaction::ref_block_commitment()`?

Reply:
Sounds good. Also renamed `block_num` -> `ref_block_num` for consistency.